packages/std/src/errors/std_error.rs

```diff
@@ -526,6 +526,15 @@ impl DivideByZeroError {
     }
 }
 
+#[derive(Error, Debug, PartialEq, Eq)]
+pub enum CheckedMultiplyRatioError {
+    #[error("Denominator must not be zero")]
+    DivideByZero {},
+
+    #[error("Multiplication overflow")]
+    MulOverflow {},
```

Comment:
Let's not duplucate the enum name

```suggestion
    Overflow {},
```